CMakeLists.txt

```diff
@@ -418,18 +418,16 @@ IF (ASPECT_PRECOMPILE_HEADERS)
 ENDIF()
 
 # Check whether we can find the WorldBuilder and enable it.
-SET(ASPECT_WITH_WORLD_BUILDER ON CACHE BOOL "If ON the World Builder is compiled togheter with ASPECT.")
-IF (NOT EXISTS ${CMAKE_SOURCE_DIR}/contrib/WorldBuilder/VERSION)
-  MESSAGE("-- World Builder not found.")
-  SET(ASPECT_WITH_WORLD_BUILDER OFF CACHE BOOL "" FORCE)
+SET(WORLD_BUILDER_SOURCE_DIR "" CACHE PATH "Provide an external World Builder directory to be compiled with ASPECT. If the path is not not provided or the World Builder is not found in the provided location, the version in the contrib folder is is used.")
+IF (NOT EXISTS ${WORLD_BUILDER_SOURCE_DIR}/VERSION)
+  MESSAGE(STATUS "World Builder not found using internal version.")
+  SET(WORLD_BUILDER_SOURCE_DIR "${CMAKE_SOURCE_DIR}/contrib/world_builder/" CACHE PATH "" FORCE)
```

Comment:
I thought the path is ``WorldBuilder`` right now? Is there a reason you are changing it?

Reply:
I changed it to `world_builder` to be more consistent with the naming conventions of aspect